src/main/java/com/google/api/codegen/config/GapicProductConfig.java

```diff
@@ -90,6 +91,9 @@
   /** Returns the type of transport for the generated client. Defaults to Grpc. */
   public abstract TransportProtocol getTransportProtocol();
 
+  /** Returns the package name. */
+  public abstract GapicConfigPresence isGapicConfigProvided();
```

Comment:
Not sure if having the method name start with "is" is appropriate if the return type isn't boolean. Should this return a boolean?